bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -427,8 +427,8 @@ spec:
             spec:
               containers:
               - args:
-                - --debugLogging
```

Comment:
we may need this run running locally, consider following the existing pattern to enable it as part of the `olm.sh run`  similar to the `imagePullPolicy: Always` replacement 

https://github.com/openshift/windows-machine-config-operator/blob/4e0c06047be1484b2c8f4faa3ec7aa2222ae2fef/hack/common.sh#L245

Reply:
To be honest I forgot that script existed. That tooling is based around `operator-sdk run packagemanifests` has been deprecated for a while, and has been succeeded by `operator-sdk run bundle`

Reply:
> been deprecated for a while

Sounds like we need a Jira task to track this.